specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1,745 +1,886 @@
+                },
+                "threadId": {
+                    "description": "The chat thread id",
+                    "type": "string"
+                }
+            }
+        },
+        "AcsChatEventBaseProperties": {
+            "description": "Schema of common properties of all chat events",
+            "type": "object",
+            "properties": {
+                "recipientCommunicationIdentifier": {
+                    "description": "The communication identifier of the target user",
+                    "$ref": "../../../../../communication/data-plane/Common/stable/2021-03-07/common.json#/definitions/CommunicationIdentifierModel"
+                },
+                "transactionId": {
+                    "description": "The transaction id will be used as co-relation vector",
+                    "type": "string"
+                },
+                "threadId": {
+                    "description": "The chat thread id",
+                    "type": "string"
+                }
+            }
+        },
+        "AcsChatThreadParticipantProperties": {
+            "description": "Schema of the chat thread participant",
+            "type": "object",
+            "properties": {
+                "displayName": {
+                    "description": "The name of the user",
+                    "type": "string"
+                },
+                "participantCommunicationIdentifier": {
+                    "description": "The communication identifier of the user",
+                    "$ref": "../../../../../communication/data-plane/Common/stable/2021-03-07/common.json#/definitions/CommunicationIdentifierModel"
+                }
+            }
+        },
+        "AcsSmsEventBaseProperties": {
+            "description": "Schema of common properties of all SMS events",
+            "type": "object",
+            "properties": {
+                "messageId": {
+                    "description": "The identity of the SMS message",
+                    "type": "string"
+                },
+                "from": {
+                    "description": "The identity of SMS message sender",
+                    "type": "string"
+                },
+                "to": {
+                    "description": "The identity of SMS message receiver",
+                    "type": "string"
+                }
+            }
+        },
+        "AcsSmsDeliveryAttemptProperties": {
+            "description": "Schema for details of a delivery attempt",
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "description": "TimeStamp when delivery was attempted",
+                    "format": "date-time",
+                    "type": "string"
+                },
+                "segmentsSucceeded": {
+                    "description": "Number of segments that were successfully delivered",
+                    "type": "integer",
+                    "format": "int64"
```

Comment:
Is this a change to the service behavior, or should this always have been int64? The SDKs are already shipped with `int` properties. We'd need to add new `long` properties and hide the `int` ones.

Reply:
When I ran the tool, it said it needed to have format. so I added int64, I can change it to int.

Reply:
You can use int32 if that is correct value, or just leave the format off.